packages/core/HttpService.tsx

```diff
@@ -60,7 +60,10 @@ class HttpService {
         return await this.authGet(url, 0);
       }
       if (response.status === 401) {
-        throw new ErrorFetch('authGet error', response.status, false);
+        throw new ErrorFetch('Needs authorization', response.status, false);
+      }
+      if (response.status === 404) {
```

Comment:
404 does not mean that something is wrong with the server. 404 is a valid response for empty dataset.